docs/Components-Toggle.md

```diff
@@ -0,0 +1,179 @@
+# Toggle Component
+
+[Back to all components](./Components.md)
+
+This component is a flexible component that can be used to toggle a boolean value. It can be used to render a switch, checkbox, or radio button, and exposes style props for every element.
+
+## Props
+
+### `variant`
+
+The `variant` prop determines the type of toggle to render. It can be one of the following: `switch`, `checkbox`, or `radio`.
+
+```tsx
+<Toggle value={value} onValueChange={setValue} variant="checkbox" />
+```
+
+### `status`
+
+The `status` prop is used to determine the interactibility or style of the toggle. It can be set to `disabled` or `error`. It is `null` by default.
+
+When set to `error`, elements of the toggle will have their colors set to `colors.errorBackground` or `colors.error`.
+
+```tsx
+<Toggle value={value} onValueChange={setValue} status="disabled" />
+```
+
+### `editable`
+
+The `editable` prop determines whether the toggle is interactible. It is `true` by default.
```

Comment:
Maybe add a note that this has same exact functionality as setting `status=disabled` and can be used interchangably.